common/lib/xmodule/xmodule/course_module.py

```diff
@@ -855,6 +855,14 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    minimum_grade_credit = Float(
+        display_name=_("Minimum Grade For Credit"),
+        help=_(
+            "Set the course minimum grade in decimal between 0.0 and 1.0"
```

Comment:
The minimum grade that a learner must earn to receive credit in the course, as a decimal between 0.0 and 1.0. For example, for 75%, enter 0.75.